src/detect-engine.c

```diff
@@ -1905,11 +1905,9 @@ int DetectEngineBufferTypeGetByIdTransforms(
 }
 
 /* returns false if no match, true if match */
-static int DetectEngineInspectRulePacketMatches(
-    DetectEngineThreadCtx *det_ctx,
-    const DetectEnginePktInspectionEngine *engine,
-    const Signature *s,
-    Packet *p, uint8_t *_alert_flags)
+static inline int DetectEngineInspectRulePacketMatches(DetectEngineThreadCtx *det_ctx,
```

Comment:
What is the performance with the same PR, but not inlining these 2 functions ?

6-14% seems a lot to me for just inlining

Reply:
All of the performance gain was from inlining these two functions. The performance test was with a small number of light weight rules and these functions were called a lot.